.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
Having a label for the Dependabot PRs would be nice. For this to work, you need to create the label, then change this configuration file for Dependabot to use the label you created. Here's an [example](https://github.com/dmarcoux/ruby_templates/blob/d1a5de40b7ae57a962c7d0c0420c544dd188d8b2/.github/dependabot.yml#L18-L19).

We should also configure Dependabot to update indirect dependencies, [this is how you can do it](https://github.com/dmarcoux/ruby_templates/blob/d1a5de40b7ae57a962c7d0c0420c544dd188d8b2/.github/dependabot.yml#L20-L22).

Reply:
Oh nice, I didn't know it uses the `dependencies` label by default. Good to know :+1: I always used Dependabot with multiple package ecosystems, so I've always configured my own labels.